libs/client/cornflow_client/tests/unit/test_instance_solution_methods.py

```diff
@@ -386,6 +386,58 @@ def test_get_solver_config_pyomo(self):
         res = self.class_to_use.get_solver_config(initial_config)
         self.assertEqual(res, {"limits/time": 10, "lp/iterlim": 10})
 
+    def test_get_solver_config_with_remove_unknow(self):
+        config = dict(
+            solver="milp_solver.gurobi",
+            msg=True,
+            abs_gap=5,
+            solver_config=dict(time_limit=10 * 60, rel_gap=0.001,  additional = True)
+        )
+        res = self.class_to_use.get_solver_config(config, remove_unknown= True )
+        self.assertEqual(res, {"TimeLimit": 600, "MIPGap": 0.001 ,"MIPGapAbs": 5, "additional":True})
```

Comment:
the arguments of assertEqual are Expected, Actual in that order.

This is not a big deal, but if they are in the opposite order, and the test fail, the error message might be confusing.

Reply:
also, I think a cleaner way to write the test is:
```
...
expected = {"TimeLimit": 600, "MIPGap": 0.001 ,"MIPGapAbs": 5, "additional":True}
result = self.class_to_use.get_solver_config(config, remove_unknown= True )
msg="if remove_unknown is true, unmapped argument should be removed from config but not from solver_config"
self.assertEqual(expected, result, msg=msg)
```